src/EdFi.AnalyticsMiddleTier.DataStandard32/Equity/MSSQL/0000-View-StudentDisciplineActionDim-Create.sql

```diff
@@ -0,0 +1,40 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'equity_StudentDisciplineActionDim')
+BEGIN
+	DROP VIEW analytics.equity_StudentDisciplineActionDim
+END
+GO
+
+CREATE VIEW analytics.equity_StudentDisciplineActionDim
+AS
+	SELECT DISTINCT CONCAT (
+        DisciplineAction.DisciplineActionIdentifier
+        ,'-'
+        ,DisciplineAction.DisciplineDate
+        ,'-'
+        ,DisciplineAction.StudentUSI
+        ) AS DisciplineActionUniqueKey
+        ,CONCAT (
+            Student.StudentUniqueId
+            ,'-'
+            ,StudentSchoolAssociation.SchoolId
+            ) AS StudentSchoolKey
+        ,Student.StudentUniqueId AS StudentKey
+        ,StudentSchoolAssociation.SchoolId AS SchoolKey
+        ,Descriptor.Description AS DisciplineActionDescription
+        ,COALESCE(DisciplineActionStaff.StaffUSI,'0') AS StaffUSI
```

Comment:
If I remove the zero in the Coalesce every test passes. Are we missing one scenario? 
Same for DS3.1